lib/Rest/VstResponse.cpp

```diff
@@ -123,6 +123,24 @@ void VstResponse::addPayload(VPackBuffer<uint8_t>&& buffer,
   if (!options) {
     options = &VPackOptions::Options::Defaults;
   }
+  
+  auto handleBuffer = [this, options](VPackBuffer<uint8_t>&& buff) {
+    if (_contentType == rest::ContentType::VPACK) {
+      if (_payload.empty()) {
+        _payload = std::move(buff);
+      } else {
+        _payload.append(buff.data(), buff.size());
+      }
+    } else if (_contentType == rest::ContentType::JSON) {
+      // simon: usually we escape unicode char sequences,
+      // but JSON over VST is not consumed by node.js or browsers
+      velocypack::ByteBufferSinkImpl<uint8_t> sink(&_payload);
+      VPackDumper dumper(&sink, options);
+      dumper.dump(VPackSlice(buff.data()));
+    } else {
+      _payload = buff;
```

Comment:
I think it is safe to move here, too:
```suggestion
      _payload = std::move(buff);
```